internal/hud/server/server.go

```diff
@@ -44,3 +54,28 @@ func (s HeadsUpServer) ViewJSON(w http.ResponseWriter, req *http.Request) {
 		http.Error(w, fmt.Sprintf("Error rendering view payload: %v", err), http.StatusInternalServerError)
 	}
 }
+
+func (s HeadsUpServer) HandleAnalytics(w http.ResponseWriter, req *http.Request) {
+	if req.Method != "POST" {
+		http.Error(w, "must be POST request", http.StatusBadRequest)
+		return
+	}
+
+	var payloads []analyticsPayload
+
+	decoder := json.NewDecoder(req.Body)
+	err := decoder.Decode(&payloads)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("error parsing JSON payload: %v", err), http.StatusInternalServerError)
```

Comment:
this should probably be an http.StatusBadRequest
https://httpstatusdogs.com/400-bad-request